content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -49,6 +49,13 @@ RuntimeClass in the **node.k8s.io/v1beta1** API version will no longer be served
 * All existing persisted objects are accessible via the new API
 * No notable changes
 
+#### PodSecurityPolicy {#psp-v125}
+
+PodSecurityPolicy in the **policy/v1beta1** API version will no longer be served in v1.25, and the PodSecurityPolicy admission controller will be removed.
+
+PodSecurityPolicy replacements are still under discussion, but current use can be migrated to
```

Comment:
I would avoid putting uncertain future-looking statements in this doc

```suggestion
Current use can be migrated to
```

Reply:
We can link to the blog article from https://github.com/kubernetes/website/pull/27369, which I believe is due to merge and publish _before_ the v1.21 release. Usually we don't link to blog articles for information about features but in this case I (confidently) think that's  a good idea.